SWTError: No more handles [gtk_init_check() failed]

Arch Linux x86_64
Flatpak 1.8.1

user@archlinux ~> flatpak run -v com.syntevo.SmartGit
F: No installations directory in /etc/flatpak/installations.d. Skipping
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/user/.local/share/flatpak
F: Opening user flatpak installation at path /home/user/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Opening user flatpak installation at path /home/user/.local/share/flatpak
F: Opening system flatpak installation at path /var/lib/flatpak
F: Cleaning up unused container id 2801832527
F: Allocated instance id 2615880716
F: Add defaults in dir /com/syntevo/SmartGit/
F: Add locks in dir /com/syntevo/SmartGit/
F: Allowing dri access
F: Allowing host-fs access
F: Allowing wayland access
F: Allowing x11 access
F: Allowing session-dbus access
F: Allowing system-dbus access
F: Running 'bwrap --args 33 xdg-dbus-proxy --args=35'
F: Running 'bwrap --args 33 smartgit'
No protocol specified
Error invoking SmartGit
java.lang.reflect.InvocationTargetException
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.base/java.lang.reflect.Method.invoke(Method.java:566)
    at com.syntevo.QBootLoader.main(SourceFile:111)
Caused by: org.eclipse.swt.SWTError: No more handles [gtk_init_check() failed]
    at org.eclipse.swt.SWT.error(SWT.java:4750)
    at org.eclipse.swt.widgets.Display.createDisplay(Display.java:1130)
    at org.eclipse.swt.widgets.Display.create(Display.java:1070)
    at org.eclipse.swt.graphics.Device.<init>(Device.java:175)
    at org.eclipse.swt.widgets.Display.<init>(Display.java:636)
    at org.eclipse.swt.widgets.Display.<init>(Display.java:627)
    at smartgit.Wy.a(SourceFile:117)
    at com.syntevo.sc.application.i.a(SourceFile:148)
    at com.syntevo.smartgit.n.a(SourceFile:321)
    at com.syntevo.smartgit.n.a(SourceFile:248)
    at com.syntevo.sc.application.F.b(SourceFile:69)
    at com.syntevo.smartgit.SmartGit.main(SourceFile:11)

--- a/com.syntevo.SmartGit.yml
+++ b/com.syntevo.SmartGit.yml
@@ -13,7 +13,6 @@ finish-args:
   - --share=ipc
   - --share=network
   - --socket=ssh-auth
-  - --socket=wayland
   - --socket=x11
   - --talk-name=org.freedesktop.Notifications
   - --talk-name=org.freedesktop.secrets
@@ -36,12 +35,14 @@ modules:
       - --enable-valgrind=no
     sources:
       - type: archive
-        sha256: 29df50974a90987af694c0fb8926a6b366e68cacd8abd813817cfe1eb5d54524
-        url: https://download.gnome.org/sources/gcr/3.34/gcr-3.34.0.tar.xz
+        sha256: a64cc7b65757fc2cd16de1708d132a16d05cd1f62c6eba436d56fe45d4ba27e1
+        url: https://download.gnome.org/sources/gcr/3.38/gcr-3.38.0.tar.xz
+
   - name: openssh
     buildsystem: simple
     build-commands:
       - ln --symbolic /usr/bin/ssh /app/bin/ssh
+
   - name: pinentry
     cleanup:
       - "/include"
@@ -66,6 +67,7 @@ modules:
       - type: archive
         sha256: 68076686fa724a290ea49cdf0d1c0c1500907d1b759a3bcbfbec0293e8f56570
         url: https://www.gnupg.org/ftp/gcrypt/pinentry/pinentry-1.1.0.tar.bz2
+
   - name: smartgit
     buildsystem: simple
     build-commands:

--- a/entrypoint.sh
+++ b/entrypoint.sh
@@ -4,7 +4,5 @@ set -o errexit
 
 SMARTGIT_JAVA_HOME="/app/extra/smartgit/jre"
 export SMARTGIT_JAVA_HOME
-TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"
-export TMPDIR
 
 exec /app/extra/smartgit/bin/smartgit.sh "$@"

